spec/proficonf.spec.js

```diff
@@ -301,14 +301,19 @@ describe('Proficonf', () => {
             return proficonf.join();
         });
 
-        async function testCommandProxy({ command, expectedRequestPayload = undefined, functionArguments = undefined }) {
+        async function testCommandProxy({
+            command,
+            functionName = undefined,
```

Comment:
Can't get it: when we call 
`testCommandProxy({ command })`
rest of args should be `undefined` by default. What do u assign them directly for?